tests/conftest.py

```diff
@@ -610,6 +610,20 @@ def web_scan(clean_db: None) -> None:
         session.commit()
 
 
+@pytest.fixture
+def in_progress_web_scan(clean_db: None) -> None:
+    """Create a dummy web scan."""
+    with models.Database() as session:
+        scan = models.Scan(
+            title="Web Scan",
+            asset="Web",
+            progress=models.ScanProgress.IN_PROGRESS,
+            created_time=datetime.datetime.now(),
+        )
+        session.add(scan)
+        session.commit()
```

Comment:
`Scan` model already exposes a create method which handles this db session logic.

Reply:
+1